src/YTree.h

```diff
@@ -182,19 +178,35 @@ class YTree : public YSelectionWidget
     virtual YTreeItem * currentItem() = 0;
 
     /**
-     * Return item in the tree which matches path of labels or nullptr in case no
-     * item with such label was found.
-     * Accepts vector of strings which denote path to the node.
-     **/
-    YTreeItem * findItem( std::vector<std::string> & path ) const;
-
-    /**
-    * Activate the item selected in the tree. Can be used in tests to simulate user input.
+    * Activate the item selected in the tree. This can be used in automated
+    * tests to simulate user input.
     *
     * Derived classes are required to implement this.
     **/
     virtual void activate() = 0;
 
+    /**
+     * Return the item in the tree that matches path of labels or 0 if not found.
+     *
+     * 'path' is a vector of strings with the path components, e.g.
+     * ["usr", "share", "doc", "packages"].
+     **/
+    YTreeItem * findItem( std::vector<std::string> & path ) const;
```

Comment:
The `vector &` should be a `const vector &`, shouldn't it? (Which propagates to iterator -> const_iterator in the other findItem)

Reply:
Right; this could definitely do with some more constness.